graphics/transparent_surface.cpp

```diff
@@ -602,7 +602,7 @@ Common::Rect TransparentSurface::blitClip(Graphics::Surface &target, Common::Rec
  * @param bKey  the blue component of the color key
  * @param overwriteAlpha if true, all other alpha will be set fully opaque
  */
-void TransparentSurface::applyColorKey(uint8 rKey, uint8 gKey, uint8 bKey, bool overwriteAlpha) {
```

Comment:
They are actually never used, they must be a leftover of some wrong approach I took. I will remove them right now.